src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -2682,7 +2682,33 @@ private bool CheckInArgTypes(TexlNode left, TexlNode right)
                     return false;
                 }
 
-                // table in anything: not supported
+                if (_txb.Document.Properties.EnabledFeatures.IsEnhancedDelegationEnabled && typeLeft.IsTable)
+                {
+                    // Table in table: RHS must be a table with a compatible schema. No coercion is allowed.
```

Comment:
>HS must be a table with a compatible schema [](http://example.com/codeflow?start=40&length=43)

RHS must be a single column table with compatible schema.

Reply:
Updated.